mako_config/mobile-bert.config

```diff
@@ -0,0 +1,60 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file is a protobuff text format representation of a Mako
+# BenchmarkInfo message.
+#
+# For more information about text format protobuffs see: go/proto-text-format
+#
+# For more information about BenchmarkInfo and the fields it contains see
+# https://github.com/google/mako/blob/master/spec/proto/mako.proto
+#
+# This particular message is configured to be passed into a create_benchmark
+# command via the mako command line tool.
+#
+# The difference between this file and update_benchmark.config is that the
+# benchmark_key is not defined.
```

Comment:
What is update_benchmark.config? We don't have it checked in. Also this file *does* define benchmark_key. I assume you're copying from https://github.com/google/mako/blob/master/examples/benchmark_info/update_benchmark.config and https://github.com/google/mako/blob/master/examples/benchmark_info/create_benchmark.config

Please review all the copied documentation to ensure it is still relevant

Reply:
I removed the comment. I think we don't need these in a config file. They can be found in Mako's example.